src/components/Colors.js

```diff
@@ -12,5 +12,10 @@ export const COLORS = {
   GRAY_STROKE_AND_DISABLED: 'GrayStrokeAndDisabled',
   WHITE: 'White',
 
+  // Background colors
+  BRAND_DUCKEGG: 'BrandDuckegg',
+  BRAND_MOSS: 'BrandMoss',
+  BRAND_BUTTERCUP: 'BrandButtercup',
```

Comment:
These came up at some point while I was mocking out cms modules, they aren't used now but there's no reason for them not to be available as variables (i think)